Composer/packages/client/src/components/WebChat/WebChatContainer.tsx

```diff
@@ -1,117 +1,41 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
-import React from 'react';
-import ReactWebChat, { createStyleSet } from 'botframework-webchat';
-import { CommunicationColors, NeutralColors } from '@uifabric/fluent-theme';
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import { NeutralColors } from '@uifabric/fluent-theme';
+import { useRecoilValue } from 'recoil';
 
-import { ConversationService, ActivityType, ChatData } from './utils/conversationService';
-import webChatStyleOptions from './utils/webChatTheme';
+import { isWebChatPanelVisibleState, webChatEssentialsSelector, rootBotProjectIdSelector } from '../../recoilModel';
+import { BASEPATH } from '../../constants';
+import { WebChatPanel } from '../../components/WebChat/WebChatPanel';
 
-export type WebChatContainerProps = {
-  currentConversation: string;
-  activeLocale: string;
-  conversationService: ConversationService;
-  botUrl: string;
-  chatData: ChatData;
-  isDisabled: boolean;
-};
-
-const createCardActionMiddleware = () => (next) => async ({ cardAction, getSignInUrl }) => {
-  const { type, value } = cardAction;
-
-  switch (type) {
-    case 'signin': {
-      // eslint-disable-next-line security/detect-non-literal-fs-filename
-      const popup = window.open();
-      const url = await getSignInUrl();
-      if (popup) {
-        popup.location.href = url;
-      }
-      break;
-    }
-
-    case 'downloadFile':
-    //Fall through
-
-    case 'playAudio':
-    //Fall through
-
-    case 'playVideo':
-    //Fall through
-
-    case 'showImage':
-    //Fall through
-
-    case 'openUrl':
-      // eslint-disable-next-line security/detect-non-literal-fs-filename
-      window.open(value, '_blank');
-      break;
-
-    default:
-      return next({ cardAction, getSignInUrl });
-  }
-};
-
-const createActivityMiddleware = () => (next: unknown) => (...setupArgs) => (...renderArgs) => {
-  const card = setupArgs[0];
-  switch (card.activity.type) {
-    case ActivityType.Trace:
-      return false;
-
-    case ActivityType.EndOfConversation:
-      return false;
-
-    default:
-      if (typeof next === 'function') {
-        const middlewareResult = next(...setupArgs);
-        if (middlewareResult) {
-          return middlewareResult(...renderArgs);
-        }
-        return false;
-      }
-      return false;
-  }
-};
+const webChatPanelZIndex = 1;
```

Comment:
move this to the zIndex array I've defined for this purpose. The idea is that all z-indices are in the same record so they wont compete with each other.
zIndices.ts